image-index.md

```diff
@@ -2,6 +2,7 @@
 
 The image index is a higher-level manifest which points to specific [image manifests](manifest.md), ideal for one or more platforms.
 While the use of an image index is OPTIONAL for image providers, image consumers SHOULD be prepared to process them.
+Also, the top-level index (`index.json`) is a REQUIRED file in [the image layout](image-layout.md).
```

Comment:
I'd rather not have this line.  This file/section is about defining the `application/vnd.oci.image.index.v1+json` media type, and should not be about where that media type is used.

I'd also encourage dropping the previous line for the same reason.  Instead, requirements about what providers may/should/must supply and what consumers may/should/must consume should be handled in the referencing location.  For example, you can see that [`application/vnd.oci.image.index.v1+json` consumers MUST handle `application/vnd.oci.image.manifest.v1+json` in the context of `manifests`][1].  And you'd see that consumers of image-layout v1.0.0 MUST handle `application/vnd.oci.image.index.v1+json` in the context of `index.json` around [here][2] (what I think you should be adding in this PR).  And you'd see that consumers need not handle v1.0.0 image layouts [here][3] (which I think you should also be adding in this PR).

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0/image-index.md#L33-L35
[2]: https://github.com/opencontainers/image-spec/blame/v1.0.0/image-layout.md#L29
[3]: https://github.com/opencontainers/image-spec/blame/v1.0.0/image-layout.md#L1-L10